src/iuse_actor.cpp

```diff
@@ -1024,10 +1025,16 @@ bool firestarter_actor::prep_firestarter_use( const player &p, tripoint &pos )
     }
 }
 
-void firestarter_actor::resolve_firestarter_use( const player &p, const tripoint &pos )
+void firestarter_actor::resolve_firestarter_use( player &p, const tripoint &pos )
 {
     if( g->m.add_field( pos, fd_fire, 1, 10_minutes ) ) {
-        p.add_msg_if_player( _( "You successfully light a fire." ) );
+        if( !p.has_trait( trait_PYROMANIA ) ) {
+            p.add_msg_if_player( _( "You successfully light a fire." ) );
+        }
+        else {
+            p.add_msg_if_player( m_good, _( "You happily light a fire." ) );
```

Comment:
Suggest `You happily light a fire.` -> `Setting that fire felt good... really good.`